### PR TITLE
feat: create Json to xml policy

### PR DESCRIPTION
**Pull request by gcusnieux**
fix gravitee-io/issues#4561

### CODE REVIEW

**Review by ytvnr — CHANGES_REQUESTED** [review]
Some remarks

1. It could be cool to add a test that take a json file and assert on the output.
2. ⚠️ There are some mistakes in the README.adoc from previous commit, at some places you keep talking about `xml-json` (see Desxription and Example sections)

File: src/main/java/io/gravitee/policy/json2xml/transformer/CDL.java [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+ * <p>
+ * The first row usually contains the names of the columns.
+ * <p>
+ * A comma delimited list can be converted into a JSONArray of JSONObjects.
+ * The names for the elements in the JSONObjects can be taken from the names
+ * in the first row.
+ * @author JSON.org
+ * @version 2015-12-09
+ */
+public class CDL {
```
[the remark this answers is not in the document]
Comment (gcusnieux) [reply src/main/java/io/gravitee/policy/json2xml/transformer/CDL.java:61]: plugins mustn't have too many dependencies

**Review by ytvnr — APPROVED** [review]
LGTM

**Pull request merged**